src/test/java/teammates/test/cases/ui/BaseActionTest.java

```diff
@@ -324,7 +324,7 @@ protected void verifyAccessibleForStudentsOfTheSameCourse(String[] submissionPar
         StudentAttributes student1InCourse1 = data.students.get("student1InCourse1");
         
         gaeSimulation.loginAsStudent(student1InCourse1.googleId);
-        verifyCanAccess(submissionParams);
+        verifyCanAccess(addUserIdToParams(student1InCourse1.googleId, submissionParams));
```

Comment:
@tarun14110, this seems strange, can you double-check this? Userid should only be used as part of the admin's masquarade mode